opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/domain/ConsentAuthTO.java

```diff
@@ -6,97 +6,95 @@
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 
+@SuppressWarnings("LineLength")
```

Comment:
Would be nice if you put explanation in comment, why you suppress this warning and not fix it...